packages/env.cpu.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
 - pytorch-cluster
 - ase
 - e3nn>=0.5
-- numpy>=1.25.0
+- numpy<2.0.0
```

Comment:
better to have range here (I think lower than 1.25 breaks some deps like pymatgen)
- numpy >=1.25.0,<2.0.0